sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/TextAnalyticsClientJavaDocCodeSnippets.java

```diff
@@ -490,6 +498,73 @@ public void analyzeSentimentWithLanguage() {
         // END: com.azure.ai.textanalytics.TextAnalyticsClient.analyzeSentiment#String-String
     }
 
+    /**
+     * Code snippet for {@link TextAnalyticsClient#analyzeSentiment(String, boolean, String)}
+     */
+    public void analyzeSentimentWithLanguageWithOpinionMining() {
+        // BEGIN: com.azure.ai.textanalytics.TextAnalyticsClient.analyzeSentiment#String-boolean-String
+        final DocumentSentiment documentSentiment = textAnalyticsClient.analyzeSentiment(
```

Comment:
I think these examples need to be more concise seeing they get injected into javadocs esp, it can make the javdocs pretty verbose.